site/docs/4.1/assets/scss/_skiplink.scss

```diff
@@ -1,14 +1,13 @@
-// stylelint-disable selector-max-id
-
-#skippy {
+.skiplink {
   display: block;
```

Comment:
We could probably use our utils here more. Not sure what mdo prefers though.

Reply:
It would be a long list of utility classes and in the and we'll still need to add additional css for `background-color` & `outline`, that's why I preferred to keep this here.